test/Tests/src/Data/Text_Spec.enso

```diff
@@ -135,8 +135,21 @@ spec =
             Ordering.compare common_prefix+complex_letter_1+later_suffix  common_prefix+complex_letter_2+earlier_suffix . should_equal Ordering.Greater
 
         Test.specify "normalizes correctly" <|
-            'aśb'.normalize . should_equal 'as\u0301b'
-            'as\u0301b'.normalize . should_equal 'as\u0301b'
+            'aśb'.codepoints . should_equal [97, 347, 98]
+            'as\u0301b'.codepoints . should_equal [97, 115, 769, 98]
+            'aśb'.normalize.codepoints . should_equal [97, 115, 769, 98]
+            'as\u0301b'.normalize.codepoints . should_equal [97, 115, 769, 98]
+
+        Test.specify "normalizes correctly using different standard Unicode normalization modes" <|
+            ## GREEK UPSILON WITH ACUTE AND HOOK SYMBOL behaves differently with each mode.
+               See https://unicode.org/faq/normalization.html.
```

Comment:
```suggestion
               See https://unicode.org/faq/normalization.html
```

Not sure if best idea, but the `.` usually is picked up by auto URL detectors (like GitHub) and breaks the link, so in cases like this one I'd ignore the point at the end of the sentence.